src/registrar/views/application.py

```diff
@@ -473,3 +474,13 @@ def get(self, request, *args, **kwargs):
         # clean up this wizard session, because we are done with it
         del self.storage
         return render(self.request, self.template_name, context)
+
+
+class ApplicationStatus(generic.DetailView):
+    model = DomainApplication
+    template_name = "application_status.html"
+    forms = []  # type: ignore
+
+    def get_context_data(self, **kwargs):
```

Comment:
**question (non-blocking)**: It looks to me like this is not doing anything different than the superclass would do. Is there a reason that this view needs to have this method at all? If there is a reason, maybe include a comment here about why. Otherwise, this method could be removed in favor of the default.

Reply:
Removed in the newer commit. This line was there because I was trying to re-use the `forms-review` code. Thanks for pointing it out.